storage/engine/rocksdb/db.cc

```diff
@@ -115,6 +120,15 @@ DBStatus ToDBStatus(const rocksdb::Status& status) {
   return ToDBString(status.ToString());
 }
 
+DBStatus FmtStatus(const char *fmt, ...) {
```

Comment:
Added `TestMVCCComputeStatsError` which writes an mvcc metadata key with garbage for the value.